packages/babel-highlight/src/index.ts

```diff
@@ -262,6 +262,7 @@ export function getChalk(options: Options) {
  * Highlight `code`.
  */
 export default function highlight(code: string, options: Options = {}): string {
+  if (!code) return code;
   if (shouldHighlight(options)) {
```

Comment:
```suggestion
  if (shouldHighlight(options) && code) {
```

Reply:
This would look more clean.

Reply:
It suffices to check empty string only `code !== ""` because `regexp#exec` always return non-null results for empty string.

Reply:
Yeah, empty string check will be more efficient and safe.